fastddsspy_participants/test/blackbox/model/EndpointDatabaseTest.cpp

```diff
@@ -116,14 +116,18 @@ std::shared_ptr<DdsPipe> create_pipe(
     DdsPipeConfiguration ddspipe_configuration;
     ddspipe_configuration.init_enabled = true;
 
+    // Create a built-in topic to transmit endpoint information
+    ddspipe_configuration.builtin_topics.insert(
+        utils::Heritable<types::DistributedTopic>::make_heritable(
+            spy::participants::endpoint_info_topic()));
```

Comment:
A bit weird to be here, but ok.

Reply:
I agree, it's a bit weird. But if not, where would it go? We have decided that it should go on top of the DdsPipe, so it only makes sense.